app/assets/stylesheets/modules/top_bar.scss

```diff
@@ -0,0 +1,40 @@
+@import 'variables';
+
+.top-bar {
+  background: $topbar-bg-color;
+
+  .name h1 {
+    color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.